cdc/puller/ddl_puller.go

```diff
@@ -42,6 +49,107 @@ const (
 	DDLPullerTableName = "DDL_PULLER"
 )
 
+// DDLJobPuller is used to pull ddl job from TiKV.
+type DDLJobPuller interface {
+	Puller
+
+	UnmarshalDDL(rawKV *model.RawKVEntry) (*timodel.Job, error)
+}
+
+type ddlJobPullerImpl struct {
+	Puller
+
+	tableInfo *model.TableInfo
+	columnID  int64
+}
+
+func (p *ddlJobPullerImpl) UnmarshalDDL(rawKV *model.RawKVEntry) (*timodel.Job, error) {
+	return entry.ParseJob(p.tableInfo, rawKV, p.columnID)
+}
+
+func findDBByName(dbs []*timodel.DBInfo, name string) (*timodel.DBInfo, error) {
+	for _, db := range dbs {
+		if db.Name.L == name {
+			return db, nil
+		}
+	}
+
+	return nil, errors.New("create NewDDLJobPuller fail, can't find schema")
```

Comment:
Please add an error to `pkg/errors`, preferably named `ErrDDLSchemaNotFound`. This seems to be an error meaningful to the user and the support staff, so giving it an RFC code seems appropriate.

Reply:
done